app/views/layouts/application.html.erb

```diff
@@ -13,12 +13,17 @@
       <%= link_to "Home", "/" %>
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
-      <%= link_to "Cart: #{cart.total_items}", "/cart" %>
+      <%= link_to "Cart: #{cart.total_items}", "/cart" if !current_admin? %>
       <%= link_to "Register", register_path if !current_user %>
       <%= link_to "Login", login_path if !current_user %>
-      <%= link_to "View My Profile", "/profile" if current_user %>
+      <%= link_to "Profile", profile_path if current_user %>
```

Comment:
nice job with the reorganization here, will display the nav bar much more intuitively